corehq/apps/locations/resources/v0_1.py

```diff
@@ -57,13 +65,7 @@ def obj_get_list(self, bundle, **kwargs):
         include_inactive = json.loads(bundle.request.GET.get('include_inactive', 'false'))
         user = bundle.request.couch_user
         viewable = _user_locations_ids(user, project, only_editable=False)
-
-        if not parent_id:
-            locs = SQLLocation.root_locations(domain, include_inactive)
-        else:
-            parent = get_object_or_not_exist(Location, parent_id, domain)
-            locs = parent.sql_location.child_locations(include_inactive)
-
+        locs = self.child_queryset(domain, include_inactive, parent_id)
```

Comment:
sorry - I had originally made the name suggestion when I thought it was only the child location queryset being changed. since you're changing the whole logic for root locations and children it probably makes more sense to use a different name. also means you have to pass a lot of context to the function which makes me less excited about this change though I think it's still marginally cleaner than overriding the whole function